recipes/sos-r/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "sos-r" %}
+{% set version = "0.19.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ed48f4f32386a9b6a7e2c13056ce43d6ff151dc33afea5a6b96316b3af0df43b
+
+build:
+  noarch: python
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  script:
+    - {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
+    - {{ R }} -e "library('IRkernel');IRkernel::installspec()"     # [not win]
+    - {{ R }} -e \"library('IRkernel');IRkernel::installspec()\"  # [win]
```

Comment:
Are those two `installspec` lines really required? The r-irkenel conda-forge package is built in a way to already provide a kernelspec just by installation. See also https://github.com/conda-forge/r-irkernel-feedstock/blob/master/recipe/build.sh

Reply:
That will be hugely helpful because I had to move `r-base` to `host` to make `{{ R }}` available.